src/react-apps/applications/service-development/src/features/deploy/deploySagas.ts

```diff
@@ -0,0 +1,220 @@
+//                   },
+//                   {
+//                     'name': 'ServiceRepositorySettings__BaseResourceFolderContainer',
+//                     'value': '/AltinnService/',
+//                   },
+//                   {
+//                     'name': 'ServiceRepositorySettings__RepositoryLocation',
+//                     'value': '/AltinnService/',
+//                   },
+//                   {
+//                     'name': 'ServiceRepositorySettings__ForceGiteaAuthentication',
+//                     'value': 'false',
+//                   },
+//                 ],
+//                 'resources': {
+
+//                 },
+//                 'terminationMessagePath': '/dev/termination-log',
+//                 'terminationMessagePolicy': 'File',
+//                 'imagePullPolicy': 'Always',
+//               },
+//             ],
+//             'restartPolicy': 'Always',
+//             'terminationGracePeriodSeconds': 30,
+//             'dnsPolicy': 'ClusterFirst',
+//             'securityContext': {
+
+//             },
+//             'imagePullSecrets': [
+//               {
+//                 'name': 'acr-secret',
+//               },
+//             ],
+//             'schedulerName': 'default-scheduler',
+//           },
+//         },
+//         'strategy': {
+//           'type': 'RollingUpdate',
+//           'rollingUpdate': {
+//             'maxUnavailable': 1,
+//             'maxSurge': 1,
+//           },
+//         },
+//         'revisionHistoryLimit': 10,
+//         'progressDeadlineSeconds': 2147483647,
+//       },
+//       'status': {
+//         'observedGeneration': 3,
+//         'replicas': 1,
+//         'updatedReplicas': 1,
+//         'readyReplicas': 1,
+//         'availableReplicas': 1,
+//         'conditions': [
+//           {
+//             'type': 'Available',
+//             'status': 'True',
+//             'lastUpdateTime': '2019-03-15T10:12:46Z',
+//             'lastTransitionTime': '2019-03-15T10:12:46Z',
+//             'reason': 'MinimumReplicasAvailable',
+//             'message': 'Deployment has minimum availability.',
+//           },
+//         ],
+//       },
+//     },
+//   ],
+// };
+
+// const mockedMasterRepoStatus = {
+//   'name': 'master',
+//   'commit': {
+//     'id': 'b508b414aad0e4c70291883b71b924c94bb87446x',
+//     'message': 'fdsff\n',
+//     'url': '/matsgm/tjeneste_0319_1346/commit/b42fe0a18bf205765aafc0b1890867f1b8c5f644',
+//     'author': {
+//       'name': 'matsgm',
+//       'email': '@jugglingnutcase',
+//       'username': '',
+//     },
+//     'committer': {
+//       'name': 'matsgm',
+//       'email': '@jugglingnutcase',
+//       'username': '',
+//     },
+//     'verification': {
+//       'verified': false,
+//       'reason': 'gpg.error.not_signed_commit',
+//       'signature': '',
+//       'payload': '',
+//     },
+//     'timestamp': '2019-03-19T12:47:27Z',
+//   },
+// };
```

Comment:
Remove commented out code, if it's not going to be used later on.

Reply:
This will be moved to tests when tests are made.